fenix/app/src/main/java/org/mozilla/fenix/components/appstate/AppAction.kt

```diff
@@ -128,6 +129,11 @@ sealed class AppAction : Action {
      */
     data class RemoveRecentSyncedTab(val syncedTab: RecentSyncedTab) : AppAction()
 
+    /**
+     * Action indicating that the selected tab has been changed.
```

Comment:
Could we add an @param for the `tab` here? I am just looking for a description to indicate that `tab` is the newly selected tab in case it wasn't clear that is being emitted here.